user/client.go

```diff
@@ -98,6 +98,7 @@ type UpdateParams struct {
 	BackupCodes                      *[]string        `json:"backup_codes,omitempty"`
 	DeleteSelfEnabled                *bool            `json:"delete_self_enabled,omitempty"`
 	CreateOrganizationEnabled        *bool            `json:"create_organization_enabled,omitempty"`
+	CreateOrganizationsLimit         *int             `json:"create_organizations_limit,omitempty"`
```

Comment:
:question: Is support for this field coming soon or already available?

I can't find it in our docs: https://clerk.com/docs/reference/backend-api/tag/Users#operation/UpdateUser

Reply:
Not yet, it's pending on this PR https://github.com/clerk/clerk_go/pull/7124! I'll add to the readme a warning